code/modules/mining/mine_turfs.dm

```diff
@@ -1078,3 +1130,139 @@ turf/unsimulated/mineral/ChangeTurf(var/turf/N, var/tell_universe=1, var/force_l
 	if(locate(/obj/structure/lattice) in contents)
 		return BUILD_SUCCESS
 	return BUILD_FAILURE
+
+/turf/simulated/floor/asteroid //For xeno nest specials
+	name = "Asteroid"
```

Comment:
Again this is how it was before and I'm unsure why it's listed me as changing it but it is not my code and I shouldn't be expected to refactor the entirety of  mine turfs, one of the worst files in the game